docs/SubmittingPatches.md

```diff
@@ -1,33 +1,39 @@
-# Submitting Patches
+# Patch submission
 
-Development on mcuboot primarily takes place in github, at:
-https://github.com/mcu-tools/mcuboot
+Development on MCUboot takes place in the [MCUboot GitHub
+repository](https://github.com/mcu-tools/mcuboot).
 
-Changes should be submitted via github pull requests.  Each commit
-should have a Signed-off-by line for the author (and the committer, if
-that is different).  It is not necessary (or possible) to get a
-Signed-off-by from Github itself, even though some commits may be
+Changes must be submitted via GitHub pull requests. Each commit should
+have a "Signed-off-by" line mentioning the author (and the committer, if
+that is different). It is not necessary (or possible) to get a
+"Signed-off-by" from Github itself, even though some commits may be
 generated by that tool.
 
 The Signed-off-by line should be at the end of the commit text, in the
-last blank-line-separated section.  There can be multiple lines in
-this section (the format being roughly like RFC-2822).  Currently
-supported trailer lines are:
+last blank-line-separated section. There can be multiple lines in this
+section, as the format is similar to RFC-2822. Currently, the supported
+trailer lines are structured as follows:
 
-    Signed-off-by: Developer Name <devname@example.com>
+- A line that indicates that the signer agrees to the "Developer
+Certificate of Origin" located at the bottom of this page:
 
-which indicates that the signer agrees to the Developer Certificate of
-Origin below, and
+  ```
+      Signed-off-by: Developer Name <devname@example.com>
+  ```
 
-    JIRA: MCUB-1234
+- A line that associate this commit with specific JIRA tickets.
```

Comment:
we don't use jira anymore instead commiter might mention GH issue: fixes #GH_issue_number etc.
Github has instruction for issue/pr mentions in commits. See: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

Reply:
fixed